src/cdk-experimental/scrolling/virtual-for-of.ts

```diff
@@ -147,8 +147,15 @@ export class CdkVirtualForOf<T> implements CollectionViewer, DoCheck, OnDestroy
       private _differs: IterableDiffers,
       /** The virtual scrolling viewport that these items are being rendered in. */
       @SkipSelf() private _viewport: CdkVirtualScrollViewport) {
-    this.dataStream.subscribe(data => this._data = data);
-    this._viewport.renderedRangeStream.subscribe(range => this._onRenderedRangeChange(range));
+    this.dataStream.subscribe(data => {
+      this._data = data;
```

Comment:
Ben's advice is that if you're creating a `BehaviorSubject`, there's probably a better way. In this case, wouldn't you want to `combineLatest`?

Reply:
At least in the case of `_data`, I'm still going to need to save it, because the latest data value is used in other places besides just `_onRenderedDataChange`. I could do a `combineLatest` for the `renderedRamgeStream` if you guys feel strongly about it. I'd prefer to just leave it as is so it looks the same as the way I'm doing the `_data` one.

Reply:
I don't feel strongly either way

Reply:
I don't feel too strongly about it either, it just felt weird to have to save the state from inside a subscription.